### PR TITLE
Trim trailing quotes from jq output

### PR DESCRIPTION
**Pull request by WilliamVenner**
Was breaking the build script

### CODE REVIEW

File: deploy/build [hunk 1]
```diff
@@ -45,7 +45,7 @@ cargo publish --all-features --package veil-macros
 current_remote_version=''
 while [ "$DRONE_TAG" != "$current_remote_version" ]
 do
-  current_remote_version=$(curl "https://crates.io/api/v1/crates/veil-macros/$DRONE_TAG" -s | jq '.version .num')
+  current_remote_version=$(curl "https://crates.io/api/v1/crates/veil-macros/$DRONE_TAG" -s | jq '.version .num' | cut -d'"' -f2)
```
Comment (MaeIsBad) [inline deploy/build:48]: You can make jq output "raw" strings with -r 
```suggestion
  current_remote_version=$(curl "https://crates.io/api/v1/crates/veil-macros/$DRONE_TAG" -s | jq -r '.version .num')
```

**Review by MaeIsBad — CHANGES_REQUESTED** [review]

File: deploy/build (hunk 1 above)
Comment (WilliamVenner) [reply deploy/build:48]: Oh brilliant, cheers!!!

**Review by MaeIsBad — APPROVED** [review]

**Comment by MaeIsBad** [comment]
I've just realized this might actually not work, because of the trailing newline. Use `-j` instead of `-r` to make sure jq doesn't output a newline if that happens

**Comment by WilliamVenner** [comment]
Seems to not have a trailing newline from what I can see:

```
william.venner@192 veil % yeah=$(curl "https://crates.io/api/v1/crates/veil-macros/0.1.3" -s | jq -r '.version .num')
william.venner@192 veil % echo "k $yeah k"                                                                           
k 0.1.3 k
```

**Comment by MaeIsBad** [comment]
Right shell scripts are funky, sorry!

**Pull request merged**